tools/uzem/uzerom.cpp

```diff
@@ -73,10 +73,30 @@ bool loadUzeImage(char* in_filename,RomHeader *header,u8 *buffer){
         if(header->version != HEADER_VERSION){
             printf("Error: cannot parse version %d UzeROM files.\n",header->version);
         }
-        printf("%s\n",header->name);
-        printf("%s\n",header->author);
-        printf("%d\n",header->year);
-        
+
+        char psupport_str[256];
+        char pdefault_str[256];

```

Comment:
```
char psupport_str[256] = {0};
char pdefault_str[256] = {0};
```
otherwise the strlen() calls below will be taking the length of uninitialized strings, which may cause out of bound writes.